Main.java

```diff
@@ -0,0 +1,33 @@
+package Task_8;
+import java.util.ArrayList;
+
+/**
+ * Class consists of a method main, which is the starting point of the program
+ *
+ * @author Bogatko Irina
+ * @version 1.0
+ * @since 12.10.2016
+ */
+public class Main {
+
+    /**
+     * Calls methods that read information from the keyboard and display its processed result
+     *
+     * @param args command-line arguments
+     */
+    public static void main(String[] args) {
+        try {
+            ArrayList<Goods> parameters = new ArrayList<>();
+
+            Input input = new Input();
+            parameters = input.getInput();
+
+            Valid validator = new Valid();
+            ArrayList<Commands> inputCommands = validator.commandsValidator();

```

Comment:
It seems to work only in "one-shot" mode, i.e. user is able to input only one command after inputting all these goods. Application must be in command mode until user enters command  "exit"